sdk/Lusid.Sdk.Tests/Features/FeatureExtractor.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace Lusid.Sdk.Tests.Features
+{
+
+    public static class FeatureExtractor
+    {
+        public static List<string> GetAllMethodAttributesInNamespace(string nameSpace)
+        {
+            var featureList = Assembly
+                .GetExecutingAssembly()
+                .GetTypes()
+                .Where(t => t.Namespace != null && t.Namespace.ToLower().StartsWith(nameSpace.ToLower()))
+                .SelectMany(t => t.GetMethods()
+                    .Where(m => m.GetCustomAttributes(typeof(LusidFeature), true)?.Length > 0)
+                    .Select(m => m.GetCustomAttributes(typeof(LusidFeature)).Cast<LusidFeature>().First().Code)
+                ).ToList();
```

Comment:
no need to do this here, returning as a List/Array will materialise the enumeration